src/server/slowlog.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <boost/circular_buffer.hpp>
+#include <string>
+#include <vector>
+
+#include "base/integral_types.h"
+#include "server/common.h"
+namespace dfly {
+
+class SlowLogEntry {
```

Comment:
I believe this can be a struct as you dont have any functionality here, just Constructor and getter functions